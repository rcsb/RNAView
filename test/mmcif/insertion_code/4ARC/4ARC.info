4ARC has one extra inserted residue after 20, 20.A
and 10 extra inserted residues after 47, 47.A through 47.J
