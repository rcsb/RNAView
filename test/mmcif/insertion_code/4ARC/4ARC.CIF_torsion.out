The backbone torsional angle for nucleic acid 

 
Note: alpha:   O3'(i-1)-P-O5'-C5'
      beta:    P-O5'-C5'-C4'
      gamma:   O5'-C5'-C4'-C3'
      delta:   C5'-C4'-C3'-O3'
      epsilon: C4'-C3'-O3'-P(i+1)
      zeta:    C3'-O3'-P(i+1)-O5'(i+1)

      chi for pyrimidines(Y): O4'-C1'-N1-C2
          chi for purines(R): O4'-C1'-N9-C4

    (The number -9999.99 means that the angle can not be calculated) 

Ch Res  Num    alpha    beta     gamma    delta  epsilon    zeta     chi
B   G    5  -9999.99   -66.11  -140.81   101.14   -62.87   -90.03   173.84
B   G    6     45.18  -104.55  -116.79    94.12  -136.92   -68.33   178.33
B   A    7    -70.04  -176.50    67.81   136.84   -99.55   -58.24  -138.36
B   U    8    -68.10  -161.82    71.59    83.81  -108.99   -79.26  -162.58
B   G    9   -176.69   111.08   175.61   148.33  -111.23   -75.63    78.04
B   G   10    173.55  -151.49    39.19    76.97  -163.82   -65.21   178.08
B   U   11    -90.47  -173.39    74.80    79.70  -150.67   -74.63  -169.86
B   G   12    -73.74   179.30    64.50    81.06  -146.55   -64.98  -159.65
B   G   13    168.90  -162.17   163.73    98.16  -157.18   -65.22  -173.38
B   A   14    -67.15  -171.00    46.83    73.40  -151.29   -66.72  -168.05
B   A   15    -58.87   177.14    43.35    80.70  -176.13   155.23  -153.44
B   U   16   -137.75   176.43    51.84   125.25  -136.86   102.27  -118.50
B   C   17   -155.41   -98.31   155.54    88.45  -141.21    67.80    86.54
B   G   18    103.63   121.52    25.40   137.62  -138.16  -100.47  -121.16
B   G   19    -71.82  -163.26    39.35   141.49   -98.06  -174.54   -75.17
B   U   20    -71.26  -128.45    62.77   138.94  -103.65   -70.52  -120.04
B   A   20    -61.82   153.60   173.60    94.36  -141.40   -77.17  -163.59
B   G   21    -56.77   162.70    56.00    88.81  -149.59   143.43  -147.01
B   A   22    -74.83   144.12    49.00    84.32  -120.30   -58.60  -178.32
B   C   23    -70.21   170.51    53.70    75.36  -145.59   -62.46  -157.15
B   A   24    -82.08   169.80    74.50    75.14  -154.91   -71.98  -166.76
B   C   25    -56.20   172.77    48.25    82.75  -148.89   -71.70  -164.55
B   A   26    -60.36   169.80    48.39    80.22  -149.64   -91.28  -165.71
B   A   27     35.51   108.55    23.70    66.66  -150.45   -64.30  -156.37
B   G   28    -59.74   161.55    44.60    67.05  -156.31   -59.45  -147.46
B   G   29    -98.03   150.97    96.50    64.76  -144.13   -80.53  -172.32
B   G   30    -52.06   173.46    47.33    76.09  -144.91   -96.36  -168.54
B   A   31    -21.66   150.93    48.91    81.57 -9999.99 -9999.99  -163.64
B   U   39  -9999.99   162.75    36.01    91.17  -153.12  -104.21  -173.24
B   C   40    -38.02   153.64    42.03    83.76  -165.08   -75.71  -148.80
B   C   41    171.82  -151.04   158.51    69.37  -107.79   -41.61   174.59
B   C   42   -109.04    86.90   166.49    79.79  -122.74   -59.73   177.28
B   U   43    -60.27  -171.50    35.22    77.63  -167.66   -52.51  -155.46
B   C   44    -87.26  -176.18    61.72    82.58  -158.51  -138.08  -153.23
B   G   45     83.03   175.75    58.79    74.29  -148.77   -64.98  -176.84
B   G   46    -82.66  -168.97    59.12    87.07  -142.33   -79.85  -168.45
B   C   47    -65.89   172.90    44.08    76.08  -149.01   -70.35  -164.70
B   G   47    -66.42   172.34    56.03    77.45  -157.47   -63.38  -168.79
B   U   47    -77.82  -170.91    64.45    80.86  -173.26   -92.30  -163.48
B   U   47   -151.25   136.36    62.18   120.10   -88.00   -68.21  -135.28
B   C   47    -87.59  -148.80    70.67   154.48   -99.68    81.74  -118.35
B   G   47     55.80  -171.79  -179.47    85.54  -154.41   -71.69    63.17
B   C   47    143.74  -116.34  -179.94    77.73  -149.35   -70.51  -170.32
B   G   47    -58.69  -179.94    42.81    87.15  -150.48   -82.64  -162.84
B   C   47    -49.63   155.46    49.97    77.78  -144.16   -67.57  -166.03
B   U   47    -67.43   164.19    62.90    76.76  -144.74   -76.18  -167.66
B   G   47    -68.36   168.29    64.40    83.61  -155.60   -91.60  -155.67
B   U   48    -58.72  -146.85    65.97   139.24  -156.71    85.35  -134.49
B   G   49     61.81   154.92    30.87    86.36  -166.07   -91.73  -175.67
B   C   50   -172.97  -121.15   130.79    74.73  -128.88   -76.30  -173.39
B   G   51    -65.27   168.78    61.20    72.52  -149.57   -60.87  -164.67
B   G   52   -112.62   164.92   103.02    67.33  -149.58   -69.90  -167.00
B   G   53    -62.27   178.72    59.84    76.30  -156.24   -63.86  -167.90
B   U   54    -66.88  -177.58    53.55    78.50  -134.96   -63.81  -146.30
B   U   55    -76.40  -178.56    59.78    75.38  -154.67   -66.00  -136.56
B   C   56    164.72   172.23    53.27    79.00  -137.67   -70.17  -171.64
B   A   57    -50.97   170.49    45.12    79.68  -152.60   -66.19  -160.66
B   A   58    -66.37  -141.00    71.83   148.19   -83.20  -179.19   -84.71
B   G   59     83.07  -166.86    63.52    90.34  -145.70   -56.35  -164.99
B   U   60    -55.39   166.12    58.50   130.23   -96.32   -71.61  -125.55
B   C   61    -89.31  -178.16    45.12    76.86  -166.71   -59.32  -168.98
B   C   62    -86.43  -154.59    56.76    80.96  -161.33   -64.11  -146.24
B   C   63    -77.60  -177.11    56.38    76.41  -153.01   -69.34  -148.72
B   G   64    -77.72   168.81    66.28    74.86  -148.71   -76.29  -164.23
B   C   65    -61.86   164.43    67.38    83.05  -144.27   -86.91  -166.66
B   U   66    -51.29   169.08    37.95    73.22  -148.82   -77.54  -163.23
B   C   67    -47.04   162.02    50.25    79.09  -171.97   -87.76  -156.90
B   C   68   -141.25  -139.23   112.13    83.77 -9999.99 -9999.99  -160.40
B   C   74  -9999.99  -176.12  -127.81   155.44   -87.58   -63.99  -107.84
B   C   75    -53.94  -168.35    37.43   152.64  -105.23    62.89  -116.14
B   A   76   -117.41   178.52    61.18    85.90 -9999.99 -9999.99   -80.29
